pull_into_place/commands/09_compare_best_designs.py

```diff
@@ -753,13 +791,30 @@ def main():
             PercentSubangstromMetric(),
             BuriedUnsatHbondMetric(),
             DunbrackScoreMetric(),
+            #PackStatScoreMetric(),
     ]
 
+    try:
+        workspace = pipeline.workspace_from_dir(args['<workspace>'])
+    except pipeline.WorkspaceNotFound:
+        raise IOError("Invalid workspace")
+
+
+
+    filter_path = os.path.join(workspace.root_dir,'filters.txt')
+    filters = []
+    with open(filter_path,"r") as file:
+        for line in file:
+            filters.append(line)
+    for record in filters:
+        filterclass = ExtraFilterHandler()
+        filterclass.get_name(record)
+        metrics.append(filterclass)
+
```

Comment:
This code is a little too complicated and detail-oriented to appear directly in the main function.  Instead, I think it should be turned into a function called `discover_quality_metrics()`.  You can see I have a comment about that, from when I was thinking about writing something like what you did.